circle.yml

```diff
@@ -5,7 +5,7 @@ deployment:
   production:
     branch: master
     heroku:
-      appname: hylo-node
+      appname: hylo-node-hitfin
```

Comment:
btw, I confirmed with CircleCI support that using an environment variable here should work